README.md

```diff
@@ -187,6 +193,10 @@ You can customize the close button using the data source call back `func mediaBr
 
 If you do not add any constraints to the button, media browser will automatically add required constraints to keep it on top right of the screen.
 
+### Customizing title
+
+Normally you can set `title` of `mediaBrowser`. You also can customize display title using delegate `func mediaBrowser(_ mediaBrowser: ATGMediaBrowser.MediaBrowserViewController, didChangeFocusTo index: Int)` to display change of item index.
```

Comment:
Last part will be better as `to display appropriate title for the changed item index`